.changeset/fifty-owls-study.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/pages-shared": patch
+---
+
+fix: Store asset key instead of body in preservation cache
```

Comment:
Ah, the cache key using the method was an additional fix not listed here.

The actual behavior of the Preservation Cache is staying the same, so maybe we should remove the changeset? We're still serving the bytes, but we retrieve them from normal storage instead of storing them in the CDN cache.

Reply:
You need a changeset if you want a new release of the package. Otherwise it assumes that there is nothing to release. If you want you could change the "type" at the start of the description to `refactor:` but the changeset type should still be "patch".

Given that I was confused by what is actually changing, I would appreciate it if you updated the changeset description to spell out what you are doing here.

Otherwise LGTM!